src/main.rs

```diff
@@ -10,8 +10,8 @@ use log::info;
 use std::env;
 use std::time::Duration;
 
-mod alexandria;
-use alexandria::protocol::{AlexandriaProtocol, PortalConfig};
+mod portalnet;
+use portalnet::protocol::{PortalConfig, PortalnetProtocol};
```

Comment:
Is it worth updating `PortalConfig` to `PortalnetConfig` to maintain consistency? It's not obvious to me that one way is better than the other - or that consistency in this case greatly improves clarity, since the intent is fairly obvious.